### DISCUSSION TITLE
Self-intro and Budibase integration with Graphical Database

### DISCUSSION THREAD

**Discussion by DMF194** [Budibase/budibase#7387] [General]
Hi all,

It is my first time getting my hands wet with Budibase, tried the cloud version last week and was blown away by this low code approach that can create good functional CRUD applications.

Just a quick question, is there any tutorial videos on how to integrate this with graphical databases like ArangoDB or Neo4j?

I have read articles on it https://budibase.com/integration/arangodb/ but not enough tutorials or use cases to see the full potential.

Welcome any suggestion on good materials for this subject

**Comment by melohagan** [comment]
Hi @DMF194 

Thanks for trying out Budibase! Unfortunately the ArangoDB connector does not appear to be functional at the moment. 

As for Neo4j, you should be able to use the REST connector to be able to query the DB:
- https://neo4j.com/docs/rest-docs/current/
- https://docs.budibase.com/docs/rest

**Comment by Shiro-Yashaa** [comment]
HI all, 
Is there anyway could I connect langflow, flowise, crew ai, etc AI tools to budibase?
Iam not a coder so I couldnt do much.
